Tasks/Ignatchik/Task_6/Task_6_5.py

```diff
@@ -0,0 +1,46 @@
+import pickle
+
+def cachee(func):
+    square = {}
+    def wraper(*args):
+        for i in args:
+            if i in square:
+                result = square[i]
+            else:
+                result = func(*args)
+                square[i] = result
+        return result
```

Comment:
this is not very good, it will work only in case of function of 1 argument, and it will return a value for the last argument if there are more then 1